frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js

```diff
@@ -444,20 +444,31 @@ function applyChartLineBarSettings(
   }
 }
 
+const BUBBLE_SIZE_INDEX = 2;
+
+const getBubbleSizeMaxDomain = (datas, seriesIndex) => {
```

Comment:
I was late to realize we use the name `datas` quite a lot.

Actually wrote a comment below about it before I grepped our codebase and saw how many there are.

Strikes me as quite bizarre, no idea if this is used for real anywhere else.